src/app/settings/WishListSettings.tsx

```diff
@@ -3,36 +3,41 @@ import { t } from 'app/i18next-t';
 import { connect } from 'react-redux';
 import { RootState } from '../store/reducers';
 import { refresh } from '../shell/refresh';
-import { clearWishLists, loadWishLists } from '../wishlists/actions';
+import { clearWishLists } from '../wishlists/actions';
 import HelpLink from '../dim-ui/HelpLink';
 import { DropzoneOptions } from 'react-dropzone';
 import FileUpload from '../dim-ui/FileUpload';
 import { wishListsEnabledSelector, loadWishListAndInfoFromIndexedDB } from '../wishlists/reducer';
 import _ from 'lodash';
-import { toWishList } from 'app/wishlists/wishlist-file';
+import { Settings } from './reducer';
+import { setSetting } from './actions';
+import { transformAndStoreWishList, fetchWishList } from 'app/wishlists/wishlist-fetch';
 
 interface StoreProps {
   wishListsEnabled: boolean;
   numWishListRolls: number;
   title?: string;
   description?: string;
+  settings: Settings;
```

Comment:
The recommended way to do this is to map to the actual value you want (`wishListSource`) instead of `settings`. That way, this only needs to re-render if `wishListSource` changes, instead of every time `settings` changes.